src/components/AllMonsters.js

```diff
@@ -2,16 +2,21 @@ import React, { useEffect, useState } from 'react'
 import styled from 'styled-components/macro'
 import saveLocally from './lib/saveLocally'
 import OneMonsterItem from './OneMonsterItem'
+import LoadingAnimation from './LoadingAnimation'
 
 export default function AllMonsters() {
   const [monsterDetails, setMonsterDetails] = useState([])
 
+  const [loading, setLoading] = useState(false)
+
   useEffect(() => {
+    localStorage.clear()
     const locallyLoadedMonsterNames = JSON.parse(
       localStorage.getItem('monsternames')
     )
```

Comment:
if you clear the localStorage locallyLoadedMonsterNames will be always be null. So you don't need the JSON.parse
```suggestion
    localStorage.clear()
    const locallyLoadedMonsterNames = null
    )
```

Reply:
good spot. That was for testing purposes. Thanks